src/runtime/database.h

```diff
@@ -80,6 +80,15 @@ struct JobReflection {
   std::vector<JobTag> tags;
 };
 
+struct JobOutput {
+  long job;
+  std::string label;
+  std::string directory;
+  std::vector<std::string> commandline;
+  std::vector<std::string> environment;
+  std::vector<std::pair<std::string, int>> outputs;
```

Comment:
I think you'll want to add this vector to JobReflection and use JobReflection instead. That way you can make this all work for everything including all the other database commands. This will ideally take the place of `stdout_payload` and `stderr_payload` and just be the properly interwoven version instead.